ci/update-gh-pages.sh

```diff
@@ -100,7 +100,10 @@ jsduck \
      --output="$DOCS_DIR/" \
      --eg-iframe=$TRAVIS_BUILD_DIR/docresources/eg-iframe.html \
      --processes=0 \
-     $TRAVIS_BUILD_DIR/src/
+     --ignore-html=debug\
+     --warnings=-extend,-link,-inheritdoc \
+     --external='ol.*' \
```

Comment:
This could replace the verbose `--external` definition that is already in `jsduck-cfg.json`.